src/Console/Document/ValidateCommand.php

```diff
@@ -0,0 +1,180 @@
+<?php
+
+/**
+ * ---------------------------------------------------------------------
+ *
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ *
+ * http://glpi-project.org
+ *
+ * @copyright 2015-2023 Teclib' and contributors.
+ * @copyright 2003-2014 by the INDEPNET Development Team.
+ * @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+namespace Glpi\Console\Document;
+
+use DBmysqlIterator;
+use Document;
+use Glpi\Console\AbstractCommand;
+use Symfony\Component\Console\Helper\ProgressBar;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+final class ValidateCommand extends AbstractCommand
+{
+    private const ERROR_MISSING_FILE = 1;
+    private const ERROR_UNEXPECTED_CONTENT = 2;
+
+    /**
+     * Keep track of errors (document id => error type)
+     * @var
+     */
+    private array $errors = [];
+
+    protected function configure()
+    {
+        parent::configure();
+
+        $this->setName('glpi:document:check_files_integrity');
+        $this->setAliases(['document:check_files_integrity']);
```

Comment:
We should probably group diagnostic/fix commands into a dedicated namespace.
```suggestion
        $this->setName('glpi:diagnostic:check_documents_integrity');
        $this->setAliases(['diagnostic:check_documents_integrity']);
```